data/src/main/java/co/nimblehq/compose/crypto/data/model/response/CoinItemResponse.kt

```diff
@@ -5,17 +5,22 @@ import com.squareup.moshi.Json
 import java.math.BigDecimal
 
 data class CoinItemResponse(
+    @Json(name = "id")
     val id: String?,
+    @Json(name = "symbol")
```

Comment:
I thought we don't need to add json name for the same field name. However if you guys want to keep this, I'm fine with it.

Reply:
@luongvo If there're some fields that need to be defined, So let's keep it to make it organized 🙏

Reply:
Agree with @Wadeewee